packages/web/src/flash/Flash.js

```diff
@@ -1,27 +1,32 @@
-import { useEffect } from 'react'
+import { useEffect, useState } from 'react'
```

Comment:
We will have to rename this from `flash/Flash` -> `Flash/Flash` since it'll be a problem for people that have case-sensitive file systems.

Reply:
@peterp Are you wanting the directory to be renamed, e.g. `-> web/src/Flash/...`?

Reply:
Ah, you know... I misunderstood the intention of this folder, it's fine the way it is.